torch/csrc/jit/cuda/cuda.h

```diff
@@ -0,0 +1,177 @@
+#include <aten/src/ATen/cuda/CUDAEvent.h>
+#include <c10/core/Device.h>
+#include <c10/cuda/CUDAStream.h>
+#include <torch/custom_class.h>
+
+namespace torch {
+namespace jit {
+
+class CUDAEvent;
+// This class is a wrapper around c10::cuda::CUDAStream.
+// It is needed because TorchBind does not support all of the argument types
+// for c10::cuda::CUDAStream. For more details, please refer to c10/cuda/CUDAStream.h.
+class CUDAStream final : public CustomClassHolder {
+ public:
+  CUDAStream(int64_t device = -1, int64_t priority = 0) {
+    constexpr int64_t PRIORITY_INDEX = 0;
+    stream_ = std::make_unique<c10::cuda::CUDAStream>(
+        c10::cuda::getStreamFromPool(priority > PRIORITY_INDEX, device));
```

Comment:
```suggestion
        c10::cuda::getStreamFromPool(priority < PRIORITY_INDEX, device));
```
If we want -1 to create a high-priority stream, shouldn't this be the other way around?

Reply:
Ok, yes it should be the other way around.